HIP-0007.md

```diff
@@ -0,0 +1,214 @@
+including [Handshake], [Namecoin], [Ethereum Name Service], [Unstoppable Domains],
+[yat], and [Butterfly Protocol]. **Every single one of these domain name systems
+needs DANE support in web browsers or their users will have no security. Without
+convenient and secure web browsing accessible to the average user, none of these
+systems will ever gain wide adoption.**
+
+Decentralized, blockchain-based naming systems offer a brand new security model and
+a brand new class of users and services, meaning that there is no "legacy" infrastructure
+to update. If we establish a high-quality standard right now, there will be no
+resistance from old systems that can refuse to upgrade, simply because there are no old
+systems yet. Therefore:
+
+1. DNSSEC is necessary for DANE, which means secure browsing to websites hosted on
+decentralized naming systems.
+2. DNSSEC is not government-controlled if the domain name root zone is secured by a
+decentralized consensus mechanism like [proof-of-work].
+3. DNSSEC can be strong if all participants agree to require high-quality cryptosystems.
+4. DANE is slow: decentralized systems are never convenient and there are difficult
+trade-offs for adoption. Simple technology like caching can help improve this lag.
+
+## Motivation
+
+We propose a secure, simplified subset of the DANE protocol that can secure the
+entire blockchain-naming ecosystem. Its properties are chosen to provide state-of-the-art
+cryptographic security and limited flexibility to reduce the technical effort required
+by web browsers to implement it.
+
+## Goals
+
+1. Do not alter or affect in any way the resolution, authentication, or fetching
+of ICANN-rooted domain names.
+
+2. When browsing to a decentralized domain name, indicate to the user whether or not
+the connection is secure (just like for legacy domains). A connection MUST
+be indicated as secure ONLY if ALL requirements of the specification are satisfied.
+
+3. Reduce the complexity (and increase the cryptographic security) of implementing
+DANE in browsers by limiting available options.
+
+## Specification
+
+### DNS Resolution
+
+This protocol does not require that browsers resolve domain names internally, although
+some networks like Handshake and Namecoin have embedable light clients: [hnsd] [ncds].
+It is expected that users will already have their system configured to resolve names
+by a trusted resolver (e.g. validating proof-of-work) that is either maintained by themselves
+or optionally, by a browser vendor for public use. Such a resolver MUST verify DNSSEC
+and ONLY set the DNS `ad` bit when DNSSEC is validated according to this specification.
+If the `ad` bit is not set, the protocol has failed and the user should be alerted that
+the connection is insecure. DNSSEC validity MUST be checked by the browser for BOTH
+the location record (`A`/`AAAA`) and the certificate matching record (`TLSA`).
```

Comment:
Location records A/AAAA don't necessarily need the AD bit. It might be easier for implementers to leave host resolution as is. They only have to care about getting a validated TLSA rrset

Reply:
thanks added in 95f2888075374bd7ccb4e5d0f1eb4b411ae5855d